catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -117,8 +117,10 @@ public CreateOperations(
   //
   public CreateResponse create(CreateRequest createRequest)
       throws IngestException, SourceUnavailableException {
+    INGEST_LOGGER.info("Resource ingest has started.");
```

Comment:
❓ Would it be useful to make this less generic? Something like `Resource ingest has started for '{}'`?

Reply:
yes, will change this